keyboards/frosty_flake/keymaps/nikchi/keymap.c

```diff
@@ -0,0 +1,148 @@
+#include "frosty_flake.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+#include "process_unicode.h"
+#include "quantum.h"
+
+#define _______ KC_TRNS
+
+void tap(uint16_t keycode){
+    register_code(keycode);
+    unregister_code(keycode);
+};
+s
```

Comment:
There's an extra `s` here. You probably pressed `s` instead of `ctrl+s` to save.